pages/newsletter/subscribed.vue

```diff
@@ -5,8 +5,9 @@ definePageMeta({
   documentDriven: false
 })
 
-useSeoMeta({
-  robots: 'noindex, nofollow'
+// disables indexing
+defineRouteRules({
```

Comment:
I'd highly recommend using the `nuxt-simple-robots` module as it will make sure this page doesn't get included in the sitemap for you.